### ISSUE TITLE
Relating date information to tracks

### ISSUE THREAD

**Issue by jdigjudy** [jfaghm/OceanEddies#19]
Hi,
  We are trying to pull seasonal data from the mat files that eddytracking software generates and having problems figuring out how to correlate the dates file with the actual eddy tracks.  Looking at the cyclonic/anticycllonic _tracks.mat files, I see the time index of the tracks data, but how does that map to the actual date?
Thank you!

**Comment by lematt1991** [comment]
If you use the `.mat` file located at `OceanEddies/tracks_viewer/SSH/dates.mat` , this will contain a `12 x 1` matrix containing dates.  The third column of each cell in the `anticyclonic_tracks.mat` file will contain an index into the `dates` matrix.  Note that the data checked in to the repository only contains data for the first 12 weeks (10/14/1992 to 12/30/1992) as noted in the README.

**Comment by jdigjudy** [comment]
Thank you!  Actually when we found the OceanEddies/eddies files (oh!!!) - we were able to map the 3rd column in the cyclonic/anticyclonic _tracks.mat files to the dates --  it just took a while to realize the files NOT in the viewer directory were where the eddies structures live.   We are using our own data, so we actually have several years going on.  On first glance, it looked like the 3rd column was "local time - as if each track started on day 1" -- but, looking down in the 10000 entry, tracks start later on as well, so the date is truly from the beginning.  Thanks again for your help!
